sky/sdk/example/mine_digger/lib/main.dart

```diff
@@ -56,18 +51,8 @@ class MineDiggerApp extends App {
   // |uiState| keeps track of the visible player progess.
   List<List<CellState>> uiState;
 
-  Game(this.app) {
-    randomSeed = 22;
-    // Colors for each mine count:
-    // 0 - none, 1 - blue, 2-green, 3-red, 4-black, 5-dark red .. etc.
-    textStyles.add(new TextStyle(color: const Color(0xFF555555), fontWeight: bold));
-    textStyles.add(new TextStyle(color: const Color(0xFF0094FF), fontWeight: bold));
-    textStyles.add(new TextStyle(color: const Color(0xFF13A023), fontWeight: bold));
-    textStyles.add(new TextStyle(color: const Color(0xFFDA1414), fontWeight: bold));
-    textStyles.add(new TextStyle(color: const Color(0xFF1E2347), fontWeight: bold));
-    textStyles.add(new TextStyle(color: const Color(0xFF7F0037), fontWeight: bold));
-    textStyles.add(new TextStyle(color: const Color(0xFFE93BE9), fontWeight: bold));
-    initialize();
+  void initState() {
+    resetGame();
```

Comment:
woops, i bet this was a bad merge.

Is there a better way to catch cases where you move code and then merge and git adds back the code that you moved in its initial location?